src/test/regress/sql/multi_multiuser.sql

```diff
@@ -393,115 +380,8 @@ SELECT result FROM run_command_on_workers($cmd$
   LIMIT 1;
 $cmd$);
 
--- super user should be the only one being able to call worker_cleanup_job_schema_cache
-SELECT worker_cleanup_job_schema_cache();
-SET ROLE full_access;
-SELECT worker_cleanup_job_schema_cache();
-SET ROLE usage_access;
-SELECT worker_cleanup_job_schema_cache();
-SET ROLE read_access;
-SELECT worker_cleanup_job_schema_cache();
-SET ROLE no_access;
-SELECT worker_cleanup_job_schema_cache();
```

Comment:
these seem like they're still relevant, since adaptive executor uses them

Reply:
This test file was using them for testing permissions of different users.  Also we use `worker_repartition_cleanup` for repartition cleanups.

Reply:
> This test file was using them for testing permissions of different users.

as long as these functions are around, making sure we don't regress on security seems important. These tests were added as part of an audit.

Reply:
c3d8ee0a16247003716f02c83d0e504fef5c8219

put all of them back, also removed the intermediate file(with jobid 42) with an extra call to prevent file leaking.